airflow/models/dag.py

```diff
@@ -3380,6 +3388,8 @@ class DagModel(Base):
     processor_subdir = Column(String(2000), nullable=True)
     # String representing the owners
     owners = Column(String(2000))
+    # Display name of the dag
+    display_name = Column(Text)
```

Comment:
This (and the TaskInstance equivalent) should have the same nullable and default parameters.

Reply:
Also this probably shouldn’t be Text; I believe this type has some performance implications in some databases. Since a DAG or task ID has limited length, we should be able to use a String instead.

Reply:
Modified accordingly